README.md

```diff
@@ -32,25 +32,13 @@ Don't forget to ⭐ this repo if you like immudb!
 
 ---
 
-immudb is a **lightweight, high-speed immutable database** with built-in cryptographic proof and verification.
-With immudb, you can track changes in sensitive data in your transactional databases and then record those changes permanently in a
-tamperproof immudb database.
-<img align="right" src="img/immudb-mascot-small.png" width="256px"/>
-
-Traditional DB transactions and logs are hard to scale and are mutable, so there is no way to know for sure if your data has been compromised.
-immudb provides **unparalleled insights** **retroactively** of changes to your sensitive data, even
-if your perimeter has been compromised. immudb guarantees immutability by using **cryptographic data structures** internally.
-This allows you to keep a persistent history of sensitive data, such as debit/credit card transactions.
+immudb is a key-value database with built-in cryptographic proof and verification. It can track changes in sensitive data and the integrity of the history will be protected by the clients, without the need to trust the server.
 
-immudb has 4 main benefits:
+Unlike blockchains, immudb can handle millions of transactions per second, and can be used both as a lightweight service or embedded in your application as a library.
```

Comment:
I'd also keep benefits 1 and 2 at least -- it helps explain immutability and verification as well as why you would want them.

"**immudb is immutable**. You can add records, but **never change or delete records**. This lets you store critical data without fear of it being changed silently. Data stored in immudb is **cryptographically coherent and verifiable**, just like blockchains, but without all the complexity. Unlike blockchains, immudb can handle millions of transactions per second, and can be used both as a lightweight service or embedded in your application as a library."

Reply:
Thanks. I brought back this benefits with this paragraph, with a small edit:

>You can add records, but never change or delete records

I think this wording confuses people, like if you can't set the same key to a new value. So I changed it to:

>You can add new versions of existing records, but never change or delete records